src/lib.c

```diff
@@ -657,7 +657,7 @@ static COLD void close_internal(Dav1dContext **const c_out, int flush) {
         free(f->lf.level);
         free(f->lf.tx_lpf_right_edge[0]);
         free(f->lf.start_of_tile_row);
-        dav1d_refmvs_clear(&f->rf);
+        dav1d_free_aligned(f->rf.r);
```

Comment:
Do you know why they removed the free of `rf.rp_proj`?  Our logic for this is now quite different, since it's using auto-derived `impl Drop`s, so I want to make sure that stays correct amidst this change.